ocs_ci/utility/deployment_openshift_logging.py

```diff
@@ -226,8 +220,10 @@ def create_instance_in_clusterlogging(sc_name=None):
             values, storage class and size details etc.
 
     """
+    num_of_worker_nodes = helpers.get_worker_nodes()
+    num_of_master_nodes = helpers.get_master_nodes()
```

Comment:
same as above

Reply:
Have done this already in line 225 please do verify```nodes_in_cluster = len(num_of_worker_nodes) + len(num_of_master_nodes)```

Reply:
same